### DISCUSSION TITLE
sequence tables not getting replicated

### DISCUSSION THREAD

**Discussion by Rammurthy5** [pgEdge/cli#35] [Q&A]
I have `pg15.3` running in two nodes (node1, node2) and subscribed to replication sets. 

How do we get sequence tables replicated?

`TableA` has BigInt autoincrement primary key.   
for instance, if the last value of primary key is `120` in `node1`, then when we insert to `node1` the auto generated value is `121` in `node1`. 
But when i try to insert the next time in `node2` in the same `TableA`, insert fails because the the sequence value it generates is `121` again. when i retry the insert, it succeeds. 

May i please request some help here ?

**Comment by luss** [comment]
Hi Ram,

We are adding support for snowflake sequences in the very near future
